.htaccess

```diff
@@ -68,13 +68,6 @@ DirectoryIndex index.php index.html index.htm
   RewriteCond %{HTTPS} on
   RewriteRule ^ - [E=protossl:s]
 
-  # Force HTTPS for site vistors.
-  # Added 2016-10-19 ME
-  RewriteCond %{HTTPS} off [OR]
-  RewriteCond %{HTTP_HOST} ^www.librariesoforegon\.org*
-  RewriteRule ^(.*)$ https://librariesoforegon.org/$1 [L,R=301]
-
```

Comment:
It looks like we didn't get any updates for `.htaccess`, so I think we can leave `.htaccess` unchanged or bring back these lines about https:
```
# Force HTTPS for site vistors.
# Added 2016-10-19 ME
RewriteCond %{HTTPS} off [OR]
RewriteCond %{HTTP_HOST} ^www.librariesoforegon\.org*
RewriteRule ^(.*)$ https://librariesoforegon.org/$1 [L,R=301]
```